### DISCUSSION TITLE
Multisite deployment - Latency constraints

### DISCUSSION THREAD

**Discussion by KKWait** [minio/minio#18566] [Q&A] [answered]
Hi community

We're planning to deploy a contry-wide S3 solution for our cloud customers. We're aiming a stack of 4 servers in each of our 3 datacenters, 100Gb/s interconnected and I'm wondering how to design it.
Can we run a "big cluster" stretched in 3 datacenters (1 global minio config with 3x4 servers) ? Does the <10ms latency generate a bottleneck ? 
Or must we deploy 1 cluster per datacenter (3 minio config with each 4 servers) with an asynchronous site replication ?

Thanks !

**Comment by klauspost** [comment, marked as answer]
@KKWait We do not do community setup/performance reviews. Please reach out on `sales@min.io` with the details you have and we can arrange to go through your setup..
